contrib/create-waku/package.json

```diff
@@ -1,5 +1,22 @@
 {
   "name": "create-waku",
   "version": "0.4.10",
-  "bin": "./cli.js"
+  "bin": "out.js",
+  "type": "commonjs",
```

Comment:
I thought your first proposition was "module". Which do you prefer?
Ah, I see it's because esbuild outputs CJS.